test/python/compiler/test_transpiler.py

```diff
@@ -2271,14 +2271,18 @@ def test_custom_dt_preserves_properties(self):
             num_qubits=5,
             basis_gates=["id", "sx", "x", "cx", "rz"],
             coupling_map=coupling_map,
-            seed=0,
+            seed=123,
```

Comment:
What was the reason for changing the seed here?

Reply:
This test requires that the "no error" transpilation doesn't accidentally choose the best qubits by itself so that it can verify there's a difference.  There's so few qubits and possibilities that it's actually quite likely for that to happen, and it just so happened that my RNG changes caused it to trigger.